src/company_gui.cpp

```diff
@@ -2275,13 +2273,8 @@ struct CompanyWindow : Window
 			}
 
 			/* Owners of company */
-			plane = SZSP_HORIZONTAL;
-			for (uint i = 0; i < lengthof(c->share_owners); i++) {
-				if (c->share_owners[i] != INVALID_COMPANY) {
-					plane = 0;
-					break;
-				}
-			}
+			auto invalid_owner = [](auto owner) { return owner == INVALID_COMPANY; };
+			plane = std::all_of(c->share_owners.begin(), c->share_owners.end(), invalid_owner) ? 0 : SZSP_HORIZONTAL;
```

Comment:
I think the value of `plane` is inverted, it should be 0 if any owner is not `INVALID_COMPANY`

Reply:
I agree; I found that that before opening the PR, as it breaks the UI... I just failed to push that fix.